src/styl/_components/_preview.styl

```diff
@@ -106,3 +110,158 @@
             left 0
             right 0
             bottom 0
+
+    &-forward // when preview is forward
+        position relative
+
+        .teaser
+            position absolute
+            left 0
+            bottom _rem(30px)
+
+            &-title
+            &-headline
+                color $color-white
+                text-shadow 0 2px 2px $color-base
+
+            &-title
+                position relative
+                z-index 1
+                font-size 1.6em
+                color $color-white
+
+                @media $breakpoints.md
+                    font-size 2.7em
+
+            &-summary
+                display none
+
+        .media
+            .media-wrap
+                &:before
+                    width 4.5em
+                    height 3.5em
+
+                    @media $breakpoints.md
+                        width 6em
+                        height 4.5em
+
+                    left 50%
+                    transform translateX(-50%)
+                    top 0
+                    margin auto
+                    background-size 60% auto
+
+                &:after
+                    content ''
+                    position absolute
+                    left 0
+                    bottom 0
+                    z-index 0
+                    width 100%
+                    height 50%
+                    background linear-gradient(0deg, rgba(0,0,0,.6) 0%, transparent 100%)
+
+        .preview
+            &-footer
+                position absolute
+                left 0
+                right 0
+                bottom 0
+
+                .counterbar
+                    background transparent
+
+                    ../
```

Comment:
Stylus parent selector

Reply:
So.. `.preview-footer .counterbar .preview-footer {}`?

Reply:
The elements in footer must be in white color.
So .preview-footer (if the content is different than counter-bar & counter-bat-item);
and  counter-bar & counter-bat-item when it exists

![image](https://user-images.githubusercontent.com/19707072/83879208-930a7080-a73d-11ea-85ae-9d8b22b679ac.png)

Reply:
So here, the `../` refers to `.counterbar`?

Reply:
No to preview-footer.
![image](https://user-images.githubusercontent.com/19707072/83879950-bf72bc80-a73e-11ea-8add-aee82b603ed2.png)

Reply:
okay, for better maintenance, what about a the white color on the parent an the color `inherit` for the children?

Reply:
Not bad idea, but if I do this, I must add many lines, because a and item inherits style
form counterbar too.
So I should write :
![image](https://user-images.githubusercontent.com/19707072/83881419-e7631f80-a740-11ea-9d23-7a09db7ba5e8.png)

instead of (what I currently have, which is shorter) :
![image](https://user-images.githubusercontent.com/19707072/83881486-09f53880-a741-11ea-8e1d-1410abbf6311.png)